components/Pages/CarePackages/CorePackageDetails/index.js

```diff
@@ -112,7 +112,7 @@ const CorePackageDetails = ({
   return (
     <div className="core-package-details brokerage">
       <BrokerageHeader />
-      <Container maxWidth="1080px" margin="0 auto" padding="0 60px">
+      <Container maxWidth="1080px" margin="0 auto" padding="0 60px 30px">
```

Comment:
I believe the bottom gap was added in other PR. Make sure that it's not duplicated when resolving conflicts